timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+applications and infrastructure are already running.
+
+If you do not have administrative access to your cloud provider account, you will need
+to work with someone from your team with sufficient permissions to:
+
+- accept VPC peering requests,
+- configure route table rules,
+- configure security group and firewall rules.
+
+<highlight type="tip">
+The Timescale Forge VPC peering feature is currently limited to AWS VPC peering only.
+</highlight>
+
+<highlight type="tip">
+Timescale Forge defaults to a limit of 3 VPCs per project. If you need more VPCs,
+you may contact support to request a quota increase.
+</highlight>
+
+<highlight type="warning">
+Once you have attached your Timescale Forge service to a VPC, it will no longer be accessible
+via the public internet. It will only be accessible via your AWS VPC which has been peered
+with your Timescale Forge VPC.
+</highlight>
+
+## Setup
+Before you begin, log in to the [Timescale Forge console](https://console.forge.timescale.com/).
+
+### Create a new VPC
+In the Timescale Forge console, click `VPC` in the left navigation bar to go to the VPC
+dashboard. You can add new VPCs here for your Timescale Forge services to attach to.
+The VPCs created here are peered with your own VPC as part of the setup process.
+
+<img class="main-content__illustration" src="vpc-dashboard.png" alt="Navigate to the VPC dashboard in the console"/>
+
+Click `Create VPC`, type a name for your new VPC, and provide an IPv4 CIDR block.
+E.G., `10.0.0.0/16` or `192.168.0.0/24`.
+Make sure that the CIDR block you choose for your VPC does not overlap with the
+cloud VPC you are using to create a peering connection. If the CIDR blocks overlap,
+the peering process will fail. You can always find the CIDR block of your AWS VPC
+from the AWS console.
+
+<img class="main-content__illustration" src="create-vpc.png" alt="Create a new Forge VPC"/>
+
+<highlight type="tip">
+VPC peering can be enabled for free during your Timescale Forge trial, but you will be
+required to enter a valid payment method in order to create a VPC (even though you
+will not yet be charged for it).
+</highlight>
+
+### Create a peering connection
+When you have created a Timescale Forge VPC, you are ready to create a peering connection
+between your Forge VPC and your cloud VPC. To do this, expand the dropdown menu by clicking
+the `VPC Peering` column on your Forge VPC.
+
+<img class="main-content__illustration" src="create-peering-connection.png" alt="Expand the VPC Peering dropdown menu and enter info"/>
+
+From this menu, enter the AWS Account ID, AWS VPC ID, and AWS region of your
+cloud VPC for the new peering connection. When you have entered the correct
+information, click `Add peering connection` to begin the peering process.
+
+This process is asynchronous, and results in a peering connection sent to your
+AWS account for you to accept. This is an important safety mechanism, never
```

Comment:
```suggestion
AWS account for you to accept. This is an important safety mechanism -- never
```

Comma feels weird here. A `--` or a `:` is more appropriate I believe

Reply:
How about an mdash (— u2014)? Pretty sure markdown parsers transform `--` into —.

Reply:
Yeah, that was what I intended